db/migrate/20161020090506_move_topologies_to_templates.rb

```diff
@@ -0,0 +1,24 @@
+class MoveTopologiesToTemplates < ActiveRecord::Migration
+  def change
```

Comment:
Minor nitpick: since most of these migrations are not reversible, `def change` should be `def up`, and there should be a `down` which raises an IrreversibleMigration:

```ruby
def up
  # ...
end

def down
  raise ActiveRecord::IrreversibleMigration
end
```

(`add_column`, `rename_column`, `create_table`, etc, are reversible; changes to data often isn't).